api/src/test/java/org/openmrs/module/mirebalais/printer/impl/ZlEmrIdCardPrinterTest.java

```diff
@@ -80,7 +75,8 @@ public void setup() throws Exception {
         loadFromInitializer(Domain.PERSON_ATTRIBUTE_TYPES, "personAttributeTypes.csv");
         loadFromInitializer(Domain.PATIENT_IDENTIFIER_TYPES, "zlIdentifierTypes.csv");
 
-        addressBundle.installAddressTemplate(); // Install address template needed for layout on id card
+        // TODO: Install address template from iniz here
+
```

Comment:
I'm going to work on fixing this and trying to get this test working, but didn't want to worry about it to get this PR in and reviewed.

Reply:
OK, I believe I have this test working again.  Pushed a fix into this PR.